### DISCUSSION TITLE
IOT Raspberry clones ?

### DISCUSSION THREAD

**Discussion by psc** [flectra-hq/flectra#36] [General]
hello all,
i start here, let's see if someone is active ;-)

Does anyone have experience with Raspberry Clones as IOT Box ?
The Raspberrys are getting harder and harder to get, so the clones would be nice.

**Comment by parthivgls** [comment]
I personally believe going via Raspberry is way too ugly hack just to sell those boxes, it is better to go cloud way, doing a cloud printing and better dual terminal screen setup.

**Reply by psc** [reply]
There are some cases where you can't do without it.
In the POS area (printers, customer displays, cash drawers, etc.) as well as in production.
Think of QS (cameras, measuring devices), or automatic barcode scanners for product tracking in production.
No IOT cloud ;-)
The servers .. always in cloud, the devices not
